crawler/attribution/entrypoints/deep-async-stack.js

```diff
@@ -0,0 +1,13 @@
+{
+    const src = `${document.currentScript.src}`;
+    let depth = 0;
+    async function deepCall () {
+        if (depth++ < 1000) {
```

Comment:
nit but in webkit for some reason this and next step throw errors that "depth" doesn't exist 🤷

Reply:
Good catch! There's some weirdness in the way closures work in webkit, I made a workaround